yaksh/views.py

```diff
@@ -1071,9 +1072,24 @@ def show_all_questions(request):
             else:
                 context["msg"] = "Please select atleast one question to test"
 
+        if request.POST.get('question_tags'):
+            question_tags = request.POST.getlist("question_tags")
+            search_tags = []
+            for tags in question_tags:
+                search_tags.extend(re.split('[; |, |\*|\n]',tags))
```

Comment:
I dont think it makes a whole lot of difference. Anyhow I have to split the string, even if the string is returned from js.